src/docs.c

```diff
@@ -69,7 +69,7 @@ int GMT_docs (void *V_API, int mode, void *args) {
 	char cmd[PATH_MAX] = {""}, view[PATH_MAX] = {""}, URL[PATH_MAX] = {""}, module[GMT_LEN64] = {""}, name[PATH_MAX] = {""}, *t = NULL, *ext = NULL;
 	const char *group = NULL, *docname = NULL;
 	char *ps_viewer = NULL;
-	static const char *known_group[2] = {"core", "other"}, *known_doc[5] = {"cookbook", "api", "tutorial", "Gallery", "gmt.conf"};
+	static const char *known_group[2] = {"core", "other"}, *known_doc[6] = {"cookbook", "api", "tutorial", "Gallery", "gmt.conf", "gmt"};
```

Comment:
The real path of the gallery is `gallery.html`. The upper case `Gallery` works because URL is case-insensitive.
```suggestion
	static const char *known_group[2] = {"core", "other"}, *known_doc[6] = {"cookbook", "api", "tutorial", "gallery", "gmt.conf", "gmt"};
```